apps/src/lib/node/ledger/shell/vote_extensions/val_set_update.rs

```diff
@@ -176,10 +157,13 @@ where
             VoteExtensionError,
         >,
     > + '_ {
+        // TODO: filter vexts whose epoch is not the current one
```

Comment:
I guess this TODO is already handled in the `validate_valset_upd_vext_and_get_it_back` calls?

Reply:
it's handled in an awkward place, I added it to `deserialize_vote_extensions`. it's because of the stale protocol txs, that cross epoch boundaries. might see if it's feasible to move that logic here, which makes more sense